src/install/memcached/windows/install-windows.mdx

```diff
@@ -0,0 +1,28 @@
+---
+componentType: default
+headingText: Configure the Memcached integration
+---
+
+1. In the Integrations directory, `C:\Program Files\New Relic\newrelic-infra\integrations.d\`, create a copy of the sample configuration file by running:
+
+```shell
+copy memcached-config.yml.sample memcached-config.yml
+```
+
+2. Edit the `memcached-config.yml` with your info. Here's a basic configuration example:
```

Comment:
these are displaying as two step 1s. this should be fixed if you just make an ordered list with 

1.
1.
1.